src/features/jserrors/aggregate/index.js

```diff
@@ -85,6 +86,7 @@ export class Aggregate extends AggregateBase {
         this.errorOnPage = true
       }
     }
+    this.harvestObservedAt = {}
```

Comment:
For brevity, If the harvest fails and retries, what is the behavior we should expect for subsequent (duplicate) errors?

Reply:
I assume that the new instances of the error will get a new timestamp for the new harvest interval, even though the harvest will also contain data from a previous harvest that failed to send.

Reply:
OK so I did some testing and decided to just let the aggregator handle the timestamp field. If there are no objections, then the timestamp reported will be that of the first instance of the error for that specific harvest interval.